.pre-commit-hooks.yaml

```diff
@@ -72,3 +72,13 @@
   language: script
   files: \.go$
   exclude: vendor\/.*$
+
+# ---------------------------------------------------------------------------------------------------------------------
+# GNU Make specific hooks
+# ---------------------------------------------------------------------------------------------------------------------
+  - id: phony-targets
+  name: phony-targets
+  description: Linter for detecting targets marked as PHONY that don't exist.
+  entry: pre_commit_hooks/make/phony-targets.sh
+  language: script
+  files: \(Makefile|.mak|.mk)$
```

Comment:
- indentation
- `Makefile.*` e.g. `Makefile-whatever`, `Makefile.docker`

Reply:
sorry, switched to vsc recently and  somehow I fucked up my YAML configuration.. my bad